wiki/People/pl.md

```diff
@@ -0,0 +1,12 @@
+# Grupy użytkowników
+
+Poniżej przedstawiono grupy użytkowników osu!, którzy swoimi działaniami wspierają rozwój gry. Większość z nich można rozpoznać poprzez kolor nazwy użytkownika na czacie lub forum, tytuł na profilu lub odznakę.
+
+- [Administracja](/wiki/People/The_Team)
+- [Moderatorzy globalni](/wiki/People/Global_Moderation_Team)
+- [Moderatorzy](/wiki/People/Language_Moderators)
+- [Zespół zapewniania jakości](/wiki/People/Quality_Assurance_Team)
+- [Nominatorzy beatmap](/wiki/People/Beatmap_Nominators)
+- [Zespół pomocy technicznej](/wiki/People/Support_Team)
```

Comment:
The name does not match the one on the page (https://osu.ppy.sh/help/wiki/People/Support_Team)

Reply:
will be changed later i guess